icgcget/cli.py

```diff
@@ -71,6 +72,20 @@ def logger_setup(logfile):
     return logger
 
 
+def check_code(client, code):
+    if code != 0:
+        logger = logging.getLogger('__log__')
+        logger.error("{} client exited with a nonzero error code {}.".format(client, code))
+        click.ClickException("Please check client output for error messages")
+
+
+def check_access(access, name):
+    if access is None:
+        logger = logging.getLogger('__log__')
```

Comment:
Do we  really need to do this in each function? My recommendation would be to set it in a global on first init. A better place would be in a class attribute, but let's hold off on that refactoring for now.